src/test/cypress/e2e/exam/test-exam/TestExamTestRun.cy.ts

```diff
@@ -0,0 +1,154 @@
+import { Exam } from 'app/entities/exam.model';
+import { CypressExamBuilder, convertCourseAfterMultiPart } from '../../../support/requests/CourseManagementRequests';
+import dayjs from 'dayjs/esm';
+import submission from '../../../fixtures/exercise/programming/build_error/submission.json';
+import { Course } from 'app/entities/course.model';
+import { generateUUID } from '../../../support/utils';
+import { EXERCISE_TYPE } from '../../../support/constants';
+import { Exercise } from 'src/test/cypress/support/pageobjects/exam/ExamParticipation';
```

Comment:
Let's be consistent here! If you access something from the support directory for example, either do a relative or an absolute import, but don't mix it. :)
Ideally, you would decide for one or the other and apply it to all imports.

Reply:
Good point, the auto import seems to be inconsistent here for some reason. I fixed it for this PR and fixed the other files in another PR, that I will open soon. Thanks for the hint!

Reply:
PR is open here: https://github.com/ls1intum/Artemis/pull/6297